earth_enterprise/SConstruct

```diff
@@ -24,6 +24,21 @@ from khEnvironment import khEnvironment
 
 EnsureSConsVersion(0, 96, 92)
 
+
+
+
+# Check for possible objects to pass on to gradle
+def gradle_options() :
+  options = ""  # Default to no option
+  if ARGUMENTS.get( 'release', '0' ) == '1' :
```

Comment:
I don't think `== '1'` is a good test to use.  E.g., a user passing `scons release=2` would be interpreted as false here, but as true in `earth_enterprise/src/SConstruct`.

You can make your code consistent by parsing the same way:

```Python
release = int(ARGUMENTS.get('release', 0))
optimize = int(ARGUMENTS.get('optimize', 0))
gprof = ARGUMENTS.get('profile', 0) or ARGUMENTS.get('gprof', 0)
sandbox = int(ARGUMENTS.get('sandbox', 0))
native_cc = not int(ARGUMENTS.get('cross_cc', 0))
```

- [ ] Interpret `scons` command-line options consistently across `scons` scripts.